File: dayThree/java/afernoonSession/ParkingLot.java
import java.util.Scanner;
import java.util.Arrays;

public class ParkingLot {

    public static void main(String... args){
        Scanner input = new Scanner(System.in);
        
        int number = input.nextInt();
        
        System.out.print(park(number));
        
        System.out.print(slot());
        
        
    }
        
    public static int[] space = new int[20];
        
    public static String park(int number){
        
        String message = "slot is taken";
        if (space[number - 1] == 0){
            space[number - 1] = 1;
            
            message = "car added successfully";
        }
        return message;
    }
    
    public static String remove(int number){
        String message = "slot is free";
        if (space[number - 1] == 1){
            space[number - 1] = 0;
            
            message = "car removed successfully";
        }
        return message;
    }
    
    public static String slot(){
        return Arrays.toString(space);
    }
    
}

File: dayThree/python/afternoonSession/parking_lot.py
space = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,]
    
def park(number):

    message = "slot is taken"
    if (space[number - 1] == 0):
        space[number - 1] = 1
        message = "car added successfully"
    
    return message


def remove(number):
    message = "slot is free"
    if (space[number - 1] == 1):
        space[number - 1] = 0;
        
        message = "car removed successfully"
    
    return message;


def slot():
    return print(space)



def option():
    print("1. enter carping slot for where you want to park.")
    print("2. enter number you want to remove parking slot.")
    print("2. enter number you want to remove parking slot.")    
active = True
    
while active:
    number = int(input(option()))
    park(number)
    slot()
    
    
    
    
    
    
    
    
    
    
    
 
